content/news/049/index.md

```diff
@@ -67,6 +67,37 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Game Updates
 
+### [Graviton - The Great Sand Simulator](http://s.team/a/2137280)
+
+![Graviton Sand Simulator Image](graviton.gif)
+
+[Graviton - The Great Sand Simulator](https://www.gravitongame.art/)
+([Discord](https://discord.gg/3MyPaDagsd),
+[Youtube](https://www.youtube.com/watch?v=X_O90KyEt8o)) by
+[@hakolao](https://github.com/hakolao/)
+is a falling sand simulator in which you can define your own interactive
+sand rules.
```

Comment:
We move links to the bottom of the section for cleanliness 👍 if you could switch it over, that would be appreciated!
```
[This blog post][graviton-blog-post]

...

[graviton-blog-post]: https://example.com
```

Reply:
(@hakolao this was about using reference-style Markdown formatting for links, the placement of them was fine)